Sources/Basics/ProgressAnimation/PercentProgressAnimation.swift

```diff
@@ -35,27 +36,42 @@ extension ProgressAnimation {
 final class RedrawingPercentProgressAnimation: ProgressAnimationProtocol {
     private let terminal: TerminalController
     private let header: String
+    private let isColorized: Bool
     private var hasDisplayedHeader = false
 
-    init(terminal: TerminalController, header: String) {
+    init(terminal: TerminalController, header: String, isColorized: Bool) {
         self.terminal = terminal
         self.header = header
+        self.isColorized = isColorized
     }
 
     /// Creates repeating string for count times.
     /// If count is negative, returns empty string.
     private func repeating(string: String, count: Int) -> String {
         return String(repeating: string, count: max(count, 0))
     }
+    
+    func isBold() -> Bool {
+        return self.isColorized
+    }
```

Comment:
Since this function is used in a single place to initialize the `let isBold` constant binding, I don't think it's worth to maintain it separately.

Reply:
I find this clarifies the intent and I actually like having this here, and it will allow us to change the "isBold()" behaviour in a single location should we ever decide to change the "bold" condition.